rockfield/inc/wpcom-colors.php

```diff
@@ -95,6 +95,91 @@
 	// Background-color lightened
 	array( '.has-background-light-background-color[class]', 'background-color', '+1' ),
 
+	/**
+	 * Grays
+	 */
+	// Text Color (Slightly less opaque color)
+	array( '.main-navigation a:hover,
+			.site-title a:hover,
+			.social-navigation a:hover,
+			.site-title:not(:empty) + .site-description:not(:empty):before,
+			#site-navigation > div > ul > li:not(:first-child):before', 'color', 0.8 ),
```

Comment:
Ahhh, good idea to use opacity here 👍

Reply:
🙇 Thanks! Since these are just hovers, that seemed to make sense.